awscli/customizations/paginate.py

```diff
@@ -223,4 +227,7 @@ def add_to_parser(self, parser):
 
     def add_to_params(self, parameters, value):
         if value is not None:
-            parameters[self.py_name] = value
+            pagination_config = parameters.get('PaginationConfig', {})
+            pagination_option = {self._serialized_name: value}
+            pagination_config.update(pagination_option)
```

Comment:
Alternatively, this is always going to just be a single item so you could just say:

```
pagination_config[self._serialized_name] = value
parameters['PaginationConfig'] = pagination_config
```